docs/get-started/install.md

```diff
@@ -2,8 +2,65 @@
 title: 'Install'
 ---
 
-Test content
+Use the Storybook CLI to install it in a single command. Run this inside your existing project’s root directory:
 
-![optimize](./optimize.gif)
+```sh
+npx sb init
+```
+<div style="background-color:#F8FAFC">
+TODO: vet this per feedback (https://github.com/storybookjs/storybook/pull/11632#discussion_r458498081)
 
-## Heading
+</div>
+
+Storybook needs to be installed into a project that is already configured. For instance with:
```

Comment:
```suggestion
Storybook needs to be installed into a project that is already setup with a framework. It will not work on an empty project. There are many ways to bootstrap an app in given framework including:
```

Reply:
Make this an expandable section. The summary should read: "`sb init` is not made for empty projects"

Reply:
Fixed with latest commit let me know if it's ok